### PR TITLE
CLIO-36773: Change the default thButton colour, add "Secondary" styling

### PR DESCRIPTION
**Pull request by craigcarlyle**

### CODE REVIEW

File: themis_components/thButton/examples/1 - Styles/html.html [hunk 1]
```diff
@@ -9,12 +9,12 @@
...
-    <th-button type="default">Default</th-button>
-    <th-button type="default" disabled>Disabled</th-button>
-    <hr>
-
     <th-button type="destroy">Destroy</th-button>
     <th-button type="destroy" disabled>Disabled</th-button>
+    <hr>
+
+    <th-button type="secondary">Secondary</th-button>
+    <th-button type="secondary" disabled>Disabled</th-button>
```
Comment (JefCurtis) [inline themis_components/thButton/examples/1 - Styles/html.html]: How does this compare to the `thDropdown` type. Do we need to update that as well?

Comment (craigcarlyle) [inline themis_components/thButton/examples/1 - Styles/html.html]: Did. See below :)

File: themis_components/thButton/examples/1 - Styles/html.html [hunk 2]
```diff
@@ -9,12 +9,12 @@
...
 
-    <th-button type="default">Default</th-button>
-    <th-button type="default" disabled>Disabled</th-button>
-    <hr>
-
     <th-button type="destroy">Destroy</th-button>
     <th-button type="destroy" disabled>Disabled</th-button>
+    <hr>
+
+    <th-button type="secondary">Secondary</th-button>
```
Comment (JefCurtis) [inline themis_components/thButton/examples/1 - Styles/html.html]: Were there no tests that check the class types to update here?

Comment (craigcarlyle) [inline themis_components/thButton/examples/1 - Styles/html.html]: Addressed.

File: themis_components/thButton/examples/1 - Styles/html.html (hunk 1 above)
Comment (JefCurtis) [inline themis_components/thButton/examples/1 - Styles/html.html]: :white_check_mark: Resolved

File: themis_components/thButton/examples/1 - Styles/html.html (hunk 2 above)
Comment (JefCurtis) [inline themis_components/thButton/examples/1 - Styles/html.html]: :white_check_mark: Resolved

**Pull request merged**